src/pages/docs/tutorials/web-midi.md

```diff
@@ -0,0 +1,349 @@
+# Web MIDI
+
+This tutorial explains the basics of controlling an Elementary Audio app with the [Web MIDI API](https://developer.mozilla.org/en-US/docs/Web/API/Web_MIDI_API). The code discussed in this tutorial is available in the MIT-licensed [elementary-webmidi-demo](https://github.com/bgins/elementary-webmidi-demo) repository.
+
+## Playing notes
+We will create a simple web app that plays notes sent by a MIDI controller. The app will:
+
+- Detect when a MIDI device is connected and disconnected
+- Provide an interface for selecting a MIDI device
+- Display the last played MIDI note and its associated frequency
+
+The app uses vanilla JavaScript and no framework to make it broadly accessible. Keep in mind this is a toy application intended to convey ideas. Your app will likely use a framework, stronger types, defensive programming, and other niceties.
+
+At a high level, the app has three parts:
+
+- A Web MIDI module for interfacing with devices
+- Audio engine and synth modules for generating audio graphs and rendering them
+- An event emitter that sends note events from Web MIDI to the audio implementation
+
+The [Web Audio API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Audio_API) ultimately renders audio because we are working in the browser.
```

Comment:
Yup, but maybe we can say "The [web-renderer](link/to/elem/web-renderer/docs) ultimately renders the audio using the Web Audio API because..." just to fit all the pieces together here?

Reply:
Yep, good idea. 💯  Updated.